examples/graphene_apps/python_worker/avalon_worker/tests/test_worker_encryption.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import logging
+import base64
+import random
+from hashlib import sha256
+import avalon_worker.worker_encryption as worker_encryption
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+
+def main(argv):
+
```

Comment:
Add comment about this test, what it is going to test.

Reply:
I need to create a ReadMe for this PR. But I'll add comment here as well